tests/remotes.py

```diff
@@ -114,31 +97,6 @@ def get_local_url():
     return get_local_storagepath()
 
 
-def get_ssh_url():
-    return "ssh://{}@127.0.0.1:22{}".format(
-        getpass.getuser(), get_local_storagepath()
-    )
```

Comment:
I did not find any usage of this, probably, because we are moving away from real server to mocked ssh server? Anyway, I didn't want to change this atm.

Reply:
@skshetry I think it is still used in tests/func/test_api.py . We are close to getting rid of it, that is true 🙂 Let's not tackle it in this PR though.

Reply:
Yep, it is still used in https://github.com/iterative/dvc/blob/028d30e22d1e1088b0e7b68df063380f76f2a818/tests/func/test_api.py#L26

I only grepped for the `get_ssh_url`/`SSH.get_url` and didn't find any. 🤦‍♂️